packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -503,22 +506,28 @@ Future<void> buildGradleApp({
   );
 
   // Call size analyzer if --analyze-size flag was provided.
-  if (buildInfo.analyzeSize != null && !globals.platform.isWindows) {
+  if (buildInfo.codeSizeDirectory != null && !globals.platform.isWindows) {
     final SizeAnalyzer sizeAnalyzer = SizeAnalyzer(
       fileSystem: globals.fs,
       logger: globals.logger,
       processUtils: ProcessUtils.instance,
     );
+    final String archName = getNameForAndroidArch(androidBuildInfo.targetArchs.single);
+    final File aotSnapshot = globals.fs.directory(buildInfo.codeSizeDirectory)
+      .childFile('snapshot.$archName.json');
+    final File precompilerTrace = globals.fs.directory(buildInfo.codeSizeDirectory)
+      .childFile('trace.$archName.json');
```

Comment:
what's this second file?

Reply:
precompiler trace file, its used for something :) @kenzieschmoll

Reply:
@kenzieschmoll is it go/dart-devtools-aot-code-size-attribution?